packages/react-dom-bindings/src/events/ReactDOMEventReplaying.js

```diff
@@ -331,34 +332,43 @@ function attemptExplicitHydrationTarget(
     }
   }
   queuedTarget.blockedOn = null;
+  if (queuedTarget.callback) {
+    queuedTarget.callback(true);
```

Comment:
The `true` here doesn't really communicate a meaningful value since there's no `false` case that resolves. If there was one, it should probably reject. I believe it's more idiomatic for a Promise that just communicates time, to resolve to `undefined`. I.e. just `callback()`

Also technically, you probably should do `const callback = queuedTarget.callback; callback()` to avoid passing `this`. Doesn't really make a difference since this is currently always wrapped in a native promise but good not to accidentally leak the internal object.